pools-v2.json

```diff
@@ -1744,9 +1744,10 @@
     "id": 161,
     "name": "Mining Squared",
     "addresses": [
-      "3GdjWJdkJhtkxRZ3Ns1LstaoHNMBW8XsvU"
+      "3GdjWJdkJhtkxRZ3Ns1LstaoHNMBW8XsvU",
+      "3AvXzTUat4p6Qf6ZLnRNvB3mDLjp3fNmjJ"
```

Comment:
fyi this address has never been used https://mempool.space/address/3AvXzTUat4p6Qf6ZLnRNvB3mDLjp3fNmjJ - not sure where you got it from

Reply:
the "3Av" address and `/bsquared/` tag were both extracted from stratum jobs provided by https://pool.bsquared.network/, but it appears this is probably not the stratum server actually being used by the pool.